packages/crayons-core/package.json

```diff
@@ -107,6 +108,7 @@
     "@popperjs/core": "^2.10.2",
     "@stencil/core": "2.9.0",
     "date-fns": "^2.28.0",
+    "libphonenumber-js": "^1.10.7",
```

Comment:
package-lock.json has been created for crayons-core package. please delete this file. Please refer to contribution guidelines for adding a dependency to a package. 

`npm install <dep> --workspace=@freshworks/crayons` from the root folder and not inside the packages/crayons-core folder.

Reply:
Fixed ✅